src/components/todoList/TodoList.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Todo } from '../todo/Todo';
+import './TodoList.css';
+
+export const TodoList = ({ todos }) => (
+  <div className="todo__list">
+    {todos.map(todo => (
+      <div className="todo__item item " key={todo.id}>
+        <Todo {...todo} />
+      </div>
```

Comment:
```suggestion
      <Todo {...todo} key={todo.id} />
```
А div перенеси внутрь компонента Todo